src/backend/marsha/settings.py

```diff
@@ -417,54 +435,24 @@ class Production(Base):
 
     ALLOWED_HOSTS = values.ListValue(None)
 
-    # For static files in production, we want to use a backend that includes a hash in
-    # the filename, that is calculated from the file content, so that browsers always
-    # get the updated version of each file.
     STATICFILES_STORAGE = values.Value(
-        "marsha.core.storage.ConfigurableManifestS3Boto3Storage"
+        "whitenoise.storage.CompressedManifestStaticFilesStorage"
     )
 
-    # The mapping between the names of the original files and the names of the files distributed
-    # by the backend is stored in a file.
-    # The best practice is to allow this manifest file's name to change for each deployment so
-    # that several versions of the app can run in parallel without interfering with each other.
-    # We make it configurable so that it can be versioned with a deployment stamp in your CI/CD:
-    STATICFILES_MANIFEST_NAME = values.Value("staticfiles.json")
-
     AWS_S3_OBJECT_PARAMETERS = {
         "Expires": "Thu, 31 Dec 2099 20:00:00 GMT",
         "CacheControl": "max-age=94608000",
     }
     AWS_BASE_NAME = values.Value("production")
 
-    # folder where static will be stored. It matches the path_pattern used
-    # in the cloudfront configuration.
-    AWS_LOCATION = Base.STATIC_URL.lstrip("/")
-
-    # pattern matching files to ignore when hashing file names and exclude from the
-    # static files manifest
-    STATIC_POSTPROCESS_IGNORE_REGEX = values.Value(r"^js\/[0-9]*\..*\.index\.js$")
-
     SESSION_COOKIE_SECURE = True
     CSRF_COOKIE_SECURE = True
 
     # pylint: disable=invalid-name
     @property
-    def ABSOLUTE_STATIC_URL(self):
+    def STATIC_URL(self):
         """Compute the absolute static url used in the lti template."""
-        return f"//{self.CLOUDFRONT_DOMAIN}{self.STATIC_URL}"
-
-    # pylint: disable=invalid-name
-    @property
-    def AWS_STATIC_BUCKET_NAME(self):
-        """AWS Static bucket name.
-
-        If this setting is set in an environment variable we use it. Otherwise
-        the value is computed with the AWS_BASE_NAME value.
-        """
-        return os.environ.get(
-            "DJANGO_AWS_STATIC_BUCKET_NAME", f"{self.AWS_BASE_NAME}-marsha-static"
-        )
+        return f"//{self.CLOUDFRONT_DOMAIN}/static/"
```

Comment:
why not keep:
```suggestion
        return f"//{self.CLOUDFRONT_DOMAIN}{self.STATIC_URL}"
```

Reply:
because I'm redefining the variable `STATIC_URL`. There is no conflict using it in the function ?

Reply:
Yes there is a conflict : 
```
 File "/app/src/backend/marsha/settings.py", line 426, in STATIC_URL
     return f"//{self.CLOUDFRONT_DOMAIN}{self.STATIC_URL}"
   [Previous line repeated 980 more times]
RecursionError: Couldn't setup configuration 'marsha.settings.Development':  maximum recursion depth exceeded
```

Reply:
Yeah yeah sorry I was surprised that STATIC_URL includes the cloudfront domain. Are you sure it's the way to do it?
When using the Boto3 backend for example it is passed via a separate setting.

Reply:
In the documentation they are defining the `STATIC_URL` setting. I tried setting `ABSOLUTE_STATIC_URL` and it doesn't work. This is maybe specific to Boto3 storage.

Reply:
http://whitenoise.evans.io/en/stable/django.html#instructions-for-amazon-cloudfront

Reply:
This settings (`ABSOLUTE_STATIC_URL`) was used in the views. I replaced it by `STATIC_URL`

Reply:
So in fact it was only used in the lti view, not by the Boto3 storage.